lisp/ess-site.el

```diff
@@ -56,20 +56,18 @@
 
```

Comment:
A lot of ess-site.el were deleted. Where did it go?  I think people might get upset by this. ess-site.el was intended as an example of configuration file.